src/Schema/Bigquery/BigquerySchemaReflection.php

```diff
@@ -26,9 +28,13 @@ public function __construct(BigQueryClient $bqClient, string $datasetName)
      */
     public function getTablesNames(): array
     {
+        $isDatasetExist = $this->bqClient->dataset($this->datasetName)->exists();
+        if ($isDatasetExist === false) {
+            throw new ReflectionException(sprintf('Dataset "%s" not found', $this->datasetName));
+        }
         $query = $this->bqClient->query(
             sprintf(
-                'SELECT * FROM %s.INFORMATION_SCHEMA.TABLES;',
+                'SELECT * FROM %s.INFORMATION_SCHEMA.TABLES WHERE `table_type` != \'VIEW\';',
```

Comment:
ak tam boli nejake views tak to vratilo aj tie tak som to osetril a pridal implementaciu pre views